src/components/Select.js

```diff
@@ -193,7 +194,7 @@ class Select extends React.Component {
     const selected = this.state.selected || this.props.selected || { displayValue: this.props.placeholderText, value: '' };
 
     return (
-      <div className='mx-select' style={Object.assign({}, this.props.style, { position: 'relative' })}>
+      <div className='mx-select' ref={this.props.selectRef} style={Object.assign({}, this.props.style, { position: 'relative' })}>
```

Comment:
Let's move this to the `div` on the next line since it already has `tabIndex` setup for localization.